integration-cli/docker_cli_run_test.go

```diff
@@ -495,7 +495,23 @@ func TestVolumesFromGetsProperMode(t *testing.T) {
 
 // Test for GH#10618
 func TestRunNoDupVolumes(t *testing.T) {
-	cmd := exec.Command(dockerBinary, "run", "-v", "/etc:/someplace", "-v", "/usr/lib:/someplace", "busybox", "echo", "hi")
+
+	bindPath1, err := ioutil.TempDir(os.TempDir(), "test1")
```

Comment:
no need for os.TempDir, you can just use `""`, ioutil.TempDir will call os.TempDir